commercetools-models/src/test/java/io/sphere/sdk/orderedit/commands/OrderEditApplyCommandIntegrationTest.java

```diff
@@ -0,0 +1,43 @@
+package io.sphere.sdk.orderedit.commands;
+
+import com.neovisionaries.i18n.CountryCode;
+import io.sphere.sdk.models.Address;
+import io.sphere.sdk.orderedit.OrderEditFixtures;
+import io.sphere.sdk.orderedits.OrderEdit;
+import io.sphere.sdk.orderedits.commands.OrderEditApplyCommand;
+import io.sphere.sdk.orderedits.commands.OrderEditUpdateCommand;
+import io.sphere.sdk.orderedits.commands.stagedactions.SetBillingAddress;
+import io.sphere.sdk.orderedits.commands.updateactions.AddStagedAction;
+import io.sphere.sdk.orderedits.expansion.OrderEditExpansionModel;
+import io.sphere.sdk.orders.Order;
+import io.sphere.sdk.orders.queries.OrderByIdGet;
+import io.sphere.sdk.test.IntegrationTest;
+import org.assertj.core.api.Assertions;
+import org.junit.Test;
+
+public class OrderEditApplyCommandIntegrationTest extends IntegrationTest {
+
+    @Test
+    public void applyOrderEdit() {
+        OrderEditFixtures.withUpdateableOrderEdit(client(), orderEdit -> {
```

Comment:
add a parameter to `withUpdateableOrderEdit ` of type `order` to make the fixture easier to use.

Reply:
added a new fixture method that accepts Order as param.